src/Altinn.Platform/Altinn.Platform.Authentication/Authentication/Controllers/AuthenticationController.cs

```diff
@@ -25,9 +27,8 @@
 using Microsoft.Extensions.Options;
 using Microsoft.IdentityModel.Tokens;
 using Microsoft.Net.Http.Headers;
-
 using Newtonsoft.Json.Linq;
-
+using JsonSerializer = System.Text.Json.JsonSerializer;
```

Comment:
Could you attempt to add a regular using instead?
```suggestion
using System.Text.Json;
```
There might be a conflict with JsonSerializer from Newtonsoft, but I hope it'll be fine as we don't import all of Newtonsoft.

Reply:
It worked to re-write this autogenerated code to a regular using. It's updated in the newest commit.